README.md

```diff
@@ -1,10 +1,9 @@
 <!--TODO(bentheelder): fill this in much more thoroughly-->
 # `kind` - `k`ubernetes `in` `d`ocker
 
-<!--testgrid test status badge, prow build badge, and go report card-->
-<!--TODO(bentheelder): switch these jobs out once we've added more CI jobs-->
-<a href="https://k8s-testgrid.appspot.com/conformance-providerless#kind,%20master%20(dev)"><img src="https://k8s-testgrid.appspot.com/q/summary/conformance-providerless/kind,%20master%20(dev)/tests_status" /></a> <a href="https://prow.k8s.io/?job=ci-kubernetes-kind-conformance">
-<img alt="Build" src="https://prow.k8s.io/badge.svg?jobs=ci-kubernetes-kind-conformance">
+<!-- prow build badge, and go report card-->
+<a href="https://prow.k8s.io/?job=ci-kind-build">
```

Comment:
@michelle192837

Reply:
Sad, but 👍

Reply:
thanks. I'll revert later, tbh most projects don't advertise e2e status in their readme though anyhow, and I have an actual post-submit build job now.